aurora/adas_files.py

```diff
@@ -108,11 +108,15 @@ def fetch_file(filename, filetype, loc):
             raise ValueError(f"Could not fetch file {filename} from ADAS!")
         
         with open(loc, "wb") as f:
-            f.write(r.content)     
-
-    if os.path.exists(adas_data_dir + filetype + os.sep + filename):
+            f.write(r.content) 
+            
+    if filename == 'none':
```

Comment:
```suggestion
    if filename == None:
```

Reply:
@odstrcilt I now understand better the issue you're trying to solve. Thanks for bringing this up!